pom.xml

```diff
@@ -1,6 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">

```

Comment:
When making a PR, try to avoid changes that aren't meaningful, like this one: they clutter the review interface and generally add noise to the process. Another common issue like this is changing code formatting or adding/removing newlines - try not to do it unless there is a reason, or the code is worth looking at anyway. You want the reviewer to be focused on the changes that actually matter, not on changes that don't.

A good practice is to review the code yourself before you ask for a review and then try to remove any accidental changes that have snuck in.